color.go

```diff
@@ -11,6 +11,27 @@ const (
 	rgb     Color = 1 << 25
 )
 
+const (
+	// Indexed terminal color constants
+	ColorBlack   vaxis.Color = 0 | 1<<24
+	ColorMaroon  vaxis.Color = 1 | 1<<24
+	ColorGreen   vaxis.Color = 2 | 1<<24
+	ColorOlive   vaxis.Color = 3 | 1<<24
+	ColorNavy    vaxis.Color = 4 | 1<<24
+	ColorPurple  vaxis.Color = 5 | 1<<24
+	ColorTeal    vaxis.Color = 6 | 1<<24
+	ColorSilver  vaxis.Color = 7 | 1<<24
+	ColorGray    vaxis.Color = 8 | 1<<24
+	ColorRed     vaxis.Color = 9 | 1<<24
+	ColorLime    vaxis.Color = 10 | 1<<24
+	ColorYellow  vaxis.Color = 11 | 1<<24
+	ColorBlue    vaxis.Color = 12 | 1<<24
+	ColorFuschia vaxis.Color = 13 | 1<<24
+	ColorAqua    vaxis.Color = 14 | 1<<24
+	ColorWhite   vaxis.Color = 15 | 1<<24
```

Comment:
Instead of hardcoded 1<<24, change to `indexed` (defined above)

Reply:
Thanks to this i had to do a compilation check and realize that this code is actually non-functional since we are already in the `vaxis` package :p